src/openforms/submissions/api/serializers.py

```diff
@@ -346,26 +350,20 @@ def notify_suspension(self, instance: Submission, email: str):
             "save_date": timezone.now(),
             "expiration_date": datetime_removed,
             "continue_url": continue_url,
+            "rendering_text": True,
```

Comment:
`rendering_text` is only used in confirmation e-mails and intended to get the plain text e-mail variant instead of HTML. This e-mail definitely needs to be HTML if possible, with a plain text fallback version.